frontend/mulighetsrommet-cms/schemas/statistikk.js

```diff
@@ -0,0 +1,42 @@
+import { ImStatsDots } from "react-icons/im";
+
+export default {
+  name: "statistikk",
+  title: "Statistikk",
+  type: "document",
+  icon: ImStatsDots,
+  fields: [
+    {
+      name: "statistikkOverskrift",
```

Comment:
Foreslår at vi heller kaller disse feltene for `overskrift`, `innhold` og `hjelpetekst`, da blir det også litt lettere å lese